modules/swagger-codegen/src/main/resources/swift3/AlamofireImplementations.mustache

```diff
@@ -76,8 +76,15 @@ open class AlamofireRequestBuilder<T>: RequestBuilder<T> {
      May be overridden by a subclass if you want to control the request
      configuration (e.g. to override the cache policy).
      */
-    open func makeRequest(manager: SessionManager, method: HTTPMethod, encoding: ParameterEncoding, headers: [String:String]) -> DataRequest {
-        return manager.request(URLString, method: method, parameters: parameters, encoding: encoding, headers: headers)
+    open func makeRequest(manager: SessionManager, method: HTTPMethod, encoding: ParameterEncoding, headers: [String:String]) throws -> DataRequest {
+        let originalRequest = try URLRequest(url: URLString, method: method, headers: headers)
+        if let encoding = encoding as? JSONEncoding {
```

Comment:
I need to do this here since ParameterEncoding protocol only nows how to take `Parameters?`. JSONEncoding has a special encode function that will take `Any?`.